### ISSUE TITLE
ИИ может наблюдать за кораблём Абдукторов 

### ISSUE THREAD

**Issue by hwat1204** [TauCetiStation/TauCetiClassic#10583]
#### Подробное описание проблемы

ИИ может наблюдать за кораблём абдукторов 

#### Что должно было произойти

Ничего

#### Что произошло на самом деле

ИИ способен следить за абдукторами

#### Как повторить

Быть ИИ, нужно что бы член экипажа на корабле абдукторов сказал что-то в радиоканал. Тыкнуть на имя для телепорта. Профит, Теперь вы можете следить за абдукторами ( возможно можно и вручную через слежку, не проверено) 

#### Дополнительная информация:

возможность слежки не пропадает при перемещении по экрану как при просмотре за боргом, выглядит так будто на корабле абдукторов есть камеры.

**Comment by LoTaRk-FroG** [comment]
Походу камера на шлеме учитывается. Либо же ты с бейджика офицера смотрел.

**Comment by pro100kotah** [comment]
> Походу камера на шлеме учитывается. Либо же ты с бейджика офицера смотрел.

Нет. Я вот почему-то забыл об этом баге и даже не писал его, в виду своей незначительности. Если нажать обсерв на члена экипажа, которого похищают абдукторы, что-то может поломаться и когда его телепортируют к себе на корабль, обсерв уйдет вместе с ним и ты будешь за ИИ видеть их кораблики. Все ближайшие четыре команды и что они делают.
Насчет тыкнуть по человеку из радио - хз, но у меня такое вроде не работало. Разве что обсерв телепортировался вместе с человеком.
